wire-ios/Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory+Account.swift

```diff
@@ -384,7 +384,7 @@ extension SettingsCellDescriptorFactory {
     }
 
     func signOutElement() -> SettingsCellDescriptorType {
-        return SettingsSignOutCellDescriptor()
+        return SettingsSignOutCellDescriptor(trackingManager: trackingManager)
```

Comment:
Why does it need the tracking manager? It seems unrelated no?

Reply:
It was needed because we were setting the disableAnalyticsSharing property inside the `private func logout(password: String? = nil)` since I just removed that it's not needed anymore. I removed the trackingManager from the descriptor as well. https://github.com/wireapp/wire-ios/commit/a8288ec6ca5c59eacd5afbfa2b32558f2ac9deac